LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
```suggestion
Copyright (c) 2024 Multi Dimensional Identity Protocol
```

I don't know if this is the right entity, but this is what we're using in kc for now